SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
ping @lsmith77 can you please give your opinion on this one? After this PR is merged, CoreBundle will be ready for a 1.3.0-RC1 release.